lib/oli_web/templates/activity/edit.html.eex

```diff
@@ -1,10 +0,0 @@
-<script type="text/javascript" src="<%= Routes.static_path(@conn, "/js/activityeditor.js") %>"></script>
```

Comment:
Looks like 9 months ago, the `activityeditor.js` build was removed, effectively breaking this page. I'm making the (hopefully reasonable) assumption that none of this is needed anymore and cleaning the template, view, and controller up.